components/builder-web/app/package-page/PackagePageComponent.ts

```diff
@@ -155,11 +155,10 @@ export class PackagePageComponent implements OnDestroy {
     }
 
     private fetchBuilds() {
-        if (this.token) {
```

Comment:
The API currently accepts a null token. If this is a private package, we'll want that token to be sure the user can access it.